### DISCUSSION TITLE
Discussions about UI

### DISCUSSION THREAD

**Discussion by NebulaTechs** [Klocman/Bulk-Crap-Uninstaller#521] [General]
When will the ui be update？

**Comment by NebulaTechs** [comment]
<img width="356" alt="屏幕截图 2023-07-15 150449" src="https://github.com/Klocman/Bulk-Crap-Uninstaller/assets/126944988/725943e6-d08e-4b66-8e64-126fc5e8a6ae">
建议在清理文件夹(包括注册表项)时允许用户展开以便于选择（如上）。

**Reply by NebulaTechs** [reply]
还有建议显示项数及大小。

**Reply by Klocman** [reply]
This seems to be the application's own uninstaller, it's not a part of BCU so I can't do much about it.

**Comment by NebulaTechs** [comment]
No it is Uninstall Tool.
